src/features/defi/components/Overview/UnderlyingAssetsMenu.tsx

```diff
@@ -0,0 +1,51 @@
+import { ChevronDownIcon } from '@chakra-ui/icons'
+import { Box, Flex, Popover, PopoverBody, PopoverContent, PopoverTrigger } from '@chakra-ui/react'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
+
+import { PairIcons } from '../PairIcons/PairIcons'
+import type { AssetWithBalance } from './Overview'
+import { UnderlyingAssetTag } from './UnderlyingAssetsTags'
+
+type UnderlyingAssetsMenuProps = {
+  underlyingAsset: AssetWithBalance
+  underlyingAssets: AssetWithBalance[]
+}
+export const UnderlyingAssetsMenu = ({
+  underlyingAsset,
+  underlyingAssets,
+}: UnderlyingAssetsMenuProps) => (
+  <Box>
+    <Popover matchWidth>
+      <PopoverTrigger>
+        <Box cursor='pointer'>
+          <UnderlyingAssetTag asset={underlyingAsset} showPercentage={false}>
```

Comment:
`showPercentage` should probably just default to false.

Reply:
Tricky one, while I do feel like showing percentage *is* the default and passing false should be the odd case, I do like the declarativeness of just passing `showPercentage` shorthand